src/utils/color-manipulator.js

```diff
@@ -170,10 +170,10 @@ export default {
       },
     };
 
-    let ratio = this.contrastRatio(background, foreground);
+    const ratio = this.contrastRatio(background, foreground);
 
-    for (let level in levels) {
-      let range = levels[level].range;
+    for (const level in levels) {
```

Comment:
Are you saying this because of how babel is transpiling it?
That safe. However `for ... of ...` require a Symbol polyfill https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=for%20(const%20level%20of%20levels)%20%7B%0A%7D%0A